lib/index.js

```diff
@@ -44,7 +45,7 @@ bedrock.events.on('bedrock.init', () => {
 
 bedrock.events.on('bedrock.start', () => {
   const {config: {'did-io': cfg}} = bedrock;
-  const didKeyDriver = didMethodKey.driver();
+  const didKeyDriver = didMethodKey.driver(cfg.methods.key);
```

Comment:
These options don't work that way:

https://github.com/digitalbazaar/did-method-key/blob/main/lib/index.js#L12

The are instead used below.

The `didMethodWeb` driver should do the same thing with the same options, I believe. However, there are additional options for the `did:web` driver like the `fetchOptions` and `allowList` that would be passed to the driver constructor. So a little more work is needed below to split these config options up for the `did:web` driver and to do the right thing.